src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -125,7 +125,7 @@ export const Reply: React.FC<ReplyProps> = props => {
         p={1}
         right={[0, null]}
         zIndex={[null, 2]}
-        position={["fixed", "static"]}
+        position={["fixed", "fixed", "fixed", "static"]}
```

Comment:
Makes the input fixed on ipad as well. Given that we're still sliding in the details  panel up til, this works well enough.